helm/cloud-provider-azure/Chart.yaml

```diff
@@ -13,4 +13,4 @@ maintainers:
 name: cloud-provider-azure
 sources:
 - https://github.com/kubernetes-sigs/cloud-provider-azure
-version: 1.23.11
+version: 1.24.0
```

Comment:
Let's skip the chart version update for now, we can queue up these changes and release them with the next batch of cloud-provider-azure releases (which will include 1.23.12)